python/seldon_core/seldon_client.py

```diff
@@ -528,6 +529,7 @@ def explain(
         headers: Dict = None,
         http_path: str = None,
         client_return_type: str = None,
+        predictor: str = None,
```

Comment:
nit (feel free to ignore): should we call the arg `predictor_name` instead of `predictor`?

Reply:
Haha I thought of the same, I think the reason I ended up goign with this is because there were no other params with the format x_name, i.e. method, gateway, namespace, etc

Reply:
There is `deployment_name` actually!

https://github.com/SeldonIO/seldon-core/blob/f79d58c429f0b709aef71cf083341a62bbcc82c5/python/seldon_core/seldon_client.py#L158